bin/minikube/launch.py

```diff
@@ -0,0 +1,1049 @@
+        None,
+        help="Password for the username. Required is --use-email-authentication is NOT set.",
+    ),
+    redis_host: str = typer.Option(
+        ..., help="Name of the host where a Redis instance is running."
+    ),
+    db_host: str = typer.Option(
+        ..., help="Name of the host where a Database instance is running."
+    ),
+    db_password: str = typer.Option(...),
+    db_user: str = typer.Option(DEFAULT_DB_NAME),
+    db_name: str = typer.Option(DEFAULT_DB_NAME),
+    is_telemetry_enabled: bool = typer.Option(
+        DEFAULT_IS_TELEMETRY_ENABLED,
+        "--enable-telemetry",
+        help="Whether or not telemetry is enabled.",
+    ),
+    is_sentry_enabled: bool = typer.Option(
+        DEFAULT_IS_SENTRY_ENABLED,
+        "--enable-sentry",
+        help="Whether or not error monitoring using Sentry is enabled.",
+    ),
+    worker_memory_request: str = typer.Option(
+        DEFAULT_WORKER_MEMORY_REQUEST,
+        help="Amount of memory to allocate to a avatar-worker pod.",
+    ),
+    api_memory_request: str = typer.Option(
+        DEFAULT_API_MEMORY_REQUEST,
+        help="Amount of memory to allocate to a avatar-api pod.",
+    ),
+    pdfgenerator_memory_request: str = typer.Option(
+        DEFAULT_PDFGENERATOR_MEMORY_REQUEST,
+        help="Amount of memory to allocate to a avatar-pdfgenerator pod.",
+    ),
+    worker_cpu_request: str = typer.Option(
+        DEFAULT_WORKER_CPU_REQUEST,
+        help="Amount of CPU to allocate to a avatar-worker pod.",
+    ),
+    api_cpu_request: str = typer.Option(
+        DEFAULT_API_CPU_REQUEST, help="Amount of CPU to allocate to a avatar-api pod."
+    ),
+    pdfgenerator_cpu_request: str = typer.Option(
+        DEFAULT_PDFGENERATOR_CPU_REQUEST,
+        help="Amount of CPU to allocate to a avatar-pdfgenerator pod.",
+    ),
+    should_upgrade_only: bool = typer.Option(
+        False,
+        "--upgrade-only",
+        help="""Whether to run 'helm upgrade' instead of 'helm install'.\n\n"""
+        """Can be useful if you forgot to change a single value and you don't want to create a brand new release.""",
+    ),
+    is_debug: bool = typer.Option(False, "--debug", help="Show verbose output."),
+) -> AvatarResult:
+    """ADVANCED. Use create-cluster if you're new to this.
+    Create the avatar component hosting the Avatar API.
+    """
+
+    if not shared_storage_path.startswith("/"):
+        raise InvalidConfig(
+            "Only absolute filesystem paths are allowed for file storage, no URL to cloud storage."
+        )
+
+    authentication = get_authentication(
+        use_email_authentication=use_email_authentication,
+        aws_mail_account_access_key_id=aws_mail_account_access_key_id,
+        aws_mail_account_secret_access_key=aws_mail_account_secret_access_key,
+        emails=email,
+        username=username,
+        password=password,
+    )
+
+    if not should_upgrade_only and (not db_host or not redis_host):
+        typer.echo(
```

Comment:
Add the values of `db_host` and `redis_host`, or be more precise.
Here you're raising if one of them is `None`, but you say both are.

Reply:
Done.